### PR TITLE
Updating UI deps and running linters

### PR DESCRIPTION
**Pull request by drduhe**
*Issue #, if available:*

*Description of changes:*
Updating various dependencies and ensuring globe runs as desired on local dev envs. Ran full workflow to ensure it behaves properly with updates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/electron-vite-react-debug.gif
Comment (RanbirAulakh) [inline public/electron-vite-react-debug.gif:1]: Not needed

File: public/electron-vite-react.gif
Comment (RanbirAulakh) [inline public/electron-vite-react.gif:1]: Should remove this gif, it's not needed

File: public/vite.svg
Comment (RanbirAulakh) [inline public/vite.svg:1]: Should remove this gif, it's not needed

File: public/electron.svg
Comment (RanbirAulakh) [inline public/electron.svg:1]: Should remove this gif, it's not needed

File: public/electron-vite.svg
Comment (RanbirAulakh) [inline public/electron-vite.svg:1]: Should remove this gif, it's not needed

File: scripts/sync_mono.sh
Comment (RanbirAulakh) [inline scripts/sync_mono.sh:1]: Should remove this file, it's not needed

Comment (drduhe) [reply scripts/sync_mono.sh:1]: Removed

File: public/electron-vite.svg
Comment (drduhe) [reply public/electron-vite.svg:1]: Removed

File: public/electron.svg
Comment (drduhe) [reply public/electron.svg:1]: Removed

File: public/vite.svg
Comment (drduhe) [reply public/vite.svg:1]: Removed

File: public/electron-vite-react.gif
Comment (drduhe) [reply public/electron-vite-react.gif:1]: Removed

File: public/electron-vite-react-debug.gif
Comment (drduhe) [reply public/electron-vite-react-debug.gif:1]: Removed

**Review by RanbirAulakh — APPROVED** [review]

**Pull request merged**